src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/RaSecondFactorProjector.php

```diff
@@ -139,8 +126,8 @@ public function applyYubikeyPossessionProvenEvent(YubikeyPossessionProvenEvent $
                 (string) $event->yubikeyPublicId,
                 $identity->id,
                 (string) $identity->institution,
-                (string) $identifyingData->commonName,
-                (string) $identifyingData->email
+                (string) $event->commonName,
```

Comment:
Might be worth implementing the types in the RaSecondFactor so we can use as much objects for as long as possible :wink: